opdracht-1.js

```diff
@@ -0,0 +1,29 @@
+//--------------------Opdracht 1--------------------//
+//-------------Opdracht 1a-------------//
+
+const tvTypeNames = inventory.map((product) => {
+    return product.name;
+});
+
+console.log(tvTypeNames);
+
+//-------------Opdracht 1b-------------//
+
+const soldOut = inventory.filter((product) => {
+    return (product.originalStock - product.sold) === 0;
+});
+
+console.log(soldOut);
+
+//-------------Opdracht 1c-------------//
+
+const tvWithAmbilight = inventory.filter((product) => {
+    return product.options.ambiLight === true;
```

Comment:
`boolean_waarde === true` is altijd gelijk aan `boolean_waarde`, dus in dit geval
```suggestion
    return product.options.ambiLight;
```